airbyte-integrations/connectors/source-s3/source_s3/utils.py

```diff
@@ -0,0 +1,46 @@
+import multiprocessing as mp
+import traceback
+from typing import Any, List
+
+import dill
+
+
+def _run_in_external_process(fn, timeout: int, max_timeout: int, logger, args:List[Any]) -> Any:
```

Comment:
if we import it somewhere then the name should be public
```suggestion
def run_in_external_process(fn, timeout: int, max_timeout: int, logger, args:List[Any]) -> Any:
```